### PR TITLE
Update notebooks

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: nb/projects/belt_sprockets/examples.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  GTD2_5mm,
+  HTD_8mm,
+  GT2_2mm,
+  GT2_3mm,
+  GT2_5mm,
+} from './belt_sprockets.nb';
+```
+
+```JavaScript
+Pulley(MXL, 20).md('MXL').view();
```
Comment (BarbourSmith) [inline nb/projects/belt_sprockets/examples.md:23]: Do we want to add a .clean() to Pulley()? It seems like having the clean version is probably what most people would want

File: nb/projects/mold/hex.md [hunk 2]
```diff
@@ -13,10 +13,7 @@ const assembled = Hexagon(50, 50, 50)
 ![Image](hex.md.0.png)
 
 ```JavaScript
-const disassembled = assembled
-  .each(to(XY()))
-  .pack()
-  .view();
+const disassembled = assembled.each(to(XY())).pack().view();
```
Comment (BarbourSmith) [inline nb/projects/mold/hex.md:16]: What does .each(to(XY())).pack() do? Does't .pack() already do the .each(to(XY)?

**Review by BarbourSmith — APPROVED** [review]
Looks great to me 😀

File: nb/projects/belt_sprockets/examples.md (hunk 1 above)
Comment (pentacular) [reply nb/projects/belt_sprockets/examples.md:23]: Yes, I think that makes sense.

File: nb/projects/mold/hex.md (hunk 2 above)
Comment (pentacular) [reply nb/projects/mold/hex.md:16]: I didn't change pack yet, probably what we want is something a little more idiomatic to replace .each(to(XY())).

**Pull request merged**